Gemfile.lock

```diff
@@ -171,6 +162,11 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rake (10.4.2)
     ref (1.0.5)
+    reform (1.2.4)
```

Comment:
I believe that `reform` version should be set to `1.2.6` (as was described in `gemfile.lock`).